### PR TITLE
enh(release): Add Centreon Web 20.10.16 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target version

- [x] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] 22.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,35 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Web
 
+### 20.10.16
+
+#### Enhancements
+
+- [Authentication] Autologin Validation reinforcement
+- [UX] Add TheWatch url to Centreon footer
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md:27]:
```suggestion
- [UX] Add TheWatch URL to Centreon footer
```

File: i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,35 @@ commerciales, vous pouvez vous rendre sur notre
...
+
+#### Bug fixes
+
+- [Authentication] Improve LDAP authentication and authorization
+- [Core] Fixed SQL request syntax error for cron with MySQL 8
+- [Install] Fixed SQL errors in upgrade process from Centreon version < 2.8.5
+- [Resources Status] Fixed the display of old downtimes in the details tab
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md:34]:
```suggestion
- [Resources Status] Fixed the display of old downtimes in the Details tab
```

File: i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,35 @@ commerciales, vous pouvez vous rendre sur notre
...
+
+#### Security Fixes
+
+- [Administration] SQL Injection on Knowledge Base configuration form
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md:38]:
```suggestion
- [Administration] SQL injection on Knowledge Base configuration form
```

File: i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md [hunk 4]
```diff
@@ -19,6 +19,35 @@ commerciales, vous pouvez vous rendre sur notre
...
+- [Administration] SQL Injections on ACL group listing
+- [Administration] SQL Injections on LDAP listing
+- [Configuration] Command path traversal resulting in RCE on command edition form
+- [Configuration] SQL Injection on export configuration
+- [Configuration] SQL Injections on SNMP traps edition form
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-20.10/releases/centreon-core.md:43]:
```suggestion
- [Configuration] SQL injections on SNMP traps edition form
```

File: versioned_docs/version-20.10/releases/centreon-core.md [hunk 5]
```diff
@@ -19,6 +19,35 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.10.16
+
+#### Enhancements
+
+- [Authentication] Autologin Validation reinforcement
+- [UX] Add TheWatch url to Centreon footer
```
Comment (cg-tw) [inline versioned_docs/version-20.10/releases/centreon-core.md:27]:
```suggestion
- [UX] Add TheWatch URL to Centreon footer
```

File: versioned_docs/version-20.10/releases/centreon-core.md [hunk 6]
```diff
@@ -19,6 +19,35 @@ If you have feature requests or want to report a bug, please go to our
...
+#### Bug fixes
+
+- [Authentication] Improve LDAP authentication and authorization
+- [Core] Fixed SQL request syntax error for cron with MySQL 8
+- [Install] Fixed SQL errors in upgrade process from Centreon version < 2.8.5
+- [Resources Status] Fixed the display of old downtimes in the details tab
+
+#### Security Fixes
+
+- [Administration] SQL Injection on Knowledge Base configuration form
```
Comment (cg-tw) [inline versioned_docs/version-20.10/releases/centreon-core.md:38]:
```suggestion
- [Administration] SQL injection on Knowledge Base configuration form
```

File: versioned_docs/version-20.10/releases/centreon-core.md [hunk 7]
```diff
@@ -19,6 +19,35 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Administration] SQL Injections on ACL group listing
```
Comment (cg-tw) [inline versioned_docs/version-20.10/releases/centreon-core.md:39]:
```suggestion
- [Administration] SQL injections on ACL group listing
```

File: versioned_docs/version-20.10/releases/centreon-core.md [hunk 8]
```diff
@@ -19,6 +19,35 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Administration] SQL Injections on LDAP listing
```
Comment (cg-tw) [inline versioned_docs/version-20.10/releases/centreon-core.md:40]:
```suggestion
- [Administration] SQL injections on LDAP listing
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Comment by tuntoja** [comment]
duplicated with https://github.com/centreon/centreon-documentation/pull/1430

**Pull request closed without merging**